service/service_test.go

```diff
@@ -4,72 +4,8 @@ import (
 	"net"
 	"reflect"
 	"testing"
-
-	"github.com/giantswarm/micrologger/microloggertest"
-	"github.com/spf13/viper"
-
-	"github.com/giantswarm/cluster-operator/flag"
 )
 
-func Test_Service_New(t *testing.T) {
```

Comment:
We dropped these tests in the other projects already as they are rather useless. Whoever things these are important, please scream.